assets/lokomotive-kubernetes/bootkube/resources/charts/pod-checkpointer/templates/daemonset.yaml

```diff
@@ -41,6 +41,10 @@ spec:
         - --lock-file=/var/run/lock/pod-checkpointer.lock
         - --kubeconfig=/etc/checkpointer/kubeconfig
         env:
+        - name: HOST_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
```

Comment:
Do we need similar change for the bootstrap kubelet?

When cluster is not running self hosted kubelet?

Reply:
No, bootstrap kubelet also listens only on node private IP: https://github.com/kinvolk/lokomotive/blob/master/assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl#L132